ansible/roles/test/files/ptftests/arista.py

```diff
@@ -52,6 +52,8 @@ def __init__(self, ip, queue, test_params, log_cb=None, login='admin', password=
         self.min_bgp_gr_timeout = int(test_params['min_bgp_gr_timeout'])
         self.reboot_type = test_params['reboot_type']
         self.bgp_v4_v6_time_diff = test_params['bgp_v4_v6_time_diff']
+        # unit: second
+        self.ssh_cmd_timeout = 10
```

Comment:
The common practice of setting a constant for class is to set a class attribute, not an instance attribute. For example:
```python
class Arista(object):
    DEBUG = False
    SSH_CMD_TIMEOUT = 10
```

Reply:
Thanks for the reminder, refined